### ISSUE TITLE
Make Project Snapshot view more digestible

### ISSUE THREAD

**Issue by alex-ketch** [stencila/hub#834]
Currently the Snapshot Details View (screenshot attached below) has a tendency to become very long and overwhelming, with desired information becoming buried.
This is especially noticeable with the long list of snapshot files, and even more so when we try to add more sections as in #818.

I propose exploring a design to try to make the various sections more discoverable and reduce the overall page height.

![Screen Shot 2020-11-16 at 18 34 40](https://user-images.githubusercontent.com/1646307/99419431-1d005d00-28ca-11eb-97bf-fd2395992a4d.png)

**Comment by nokome** [comment]
Agreed. The expanded file listing is a long standing bug which I have not got around to fixing. But yes, beyond that there is potential for alternative page layouts.
